### PR TITLE
Issue #19480: Fix duplicate messages in WIM.

### LINKED ISSUE #19480: Fix duplicate error messages in WIM
The following error messages appear to be duplicated. Opened to address findings from WS-CD-Open #25343.

- CWIMK0002E
- CWIML4516E
- CWIML4510E
- CWIML1999E
- CWIML0510W
- CWIML4541E

### PR DESCRIPTION
**Pull request by jvanhill**
fixes #19480

### CODE REVIEW

**Comment by jvanhill** [comment]
#build

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops [hunk 1]
```diff
@@ -83,7 +83,7 @@ NULL_CHECKPOINT_VALUE=CWIML4550E: The LDAP operation could not be completed. An
 NULL_CHECKPOINT_VALUE.explanation=If a repository supports change tracking as specified by the 'supportChangeLog' flag for the repository in the server.xml file, then the checkpoint passed for it in the ChangeControl should not be empty or null.
 NULL_CHECKPOINT_VALUE.useraction=Try the search again with a valid checkpoint or disable 'supportChangeLog' for that repository in the server.xml file, if the user registry adapter is not capable of change tracking.
 
-MISSING_OR_EMPTY_PASSWORD=CWIML4541E: The login operation could not be completed. The password is missing or empty.
+MISSING_OR_EMPTY_PASSWORD=CWIML4541E: The login operation could not be completed because the password was either missing or empty.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Reviewed Message: The login operation could not be completed as the password is either missing or empty.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops [hunk 2]
```diff
@@ -127,9 +127,9 @@ EXT_ID_HAS_MULTIPLE_VALUES=CWIML4528E: The search operation could not be complet
 EXT_ID_HAS_MULTIPLE_VALUES.explanation=The LDAP attribute used as the external identifier must be unique and cannot contain multiple values.
 EXT_ID_HAS_MULTIPLE_VALUES.useraction=Ensure that the correct attribute is chosen for the external identifier. If no appropriate attribute exists, the distinguished name can be used as the external identifier.
 
-INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The configured base entry definition is not valid: {0}. Configure a valid base entry definition in the server.xml file. For example: <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.explanation=The syntax for defining a base entry is <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.useraction=Fix the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: INVALID_BASE_ENTRY_DEFINITION.explanation= The specified base entry definition is invalid.

File: dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops [hunk 3]
```diff
@@ -32,8 +32,8 @@ INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION=CWIMK0001E: The user registry operation
 INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION.explanation= The entity type must be defined with an entity type name and at least one corresponding Relative Distinguished Name (RDN) property.
 INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION.useraction=Modify and correct the entity type definition in the server.xml file.
 
-INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file. For example, the syntax is <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect. The syntax for defining a base entry is <baseEntry name="..." baseDN="...."/>.
+INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops]: INVALID_BASE_ENTRY_DEFINITION.explanation= The specified base entry definition is invalid.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops [hunk 4]
```diff
@@ -127,9 +127,9 @@ EXT_ID_HAS_MULTIPLE_VALUES=CWIML4528E: The search operation could not be complet
...
+INVALID_BASE_ENTRY_DEFINITION.useraction= Modify and correct the base entry definition in the server.xml file.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: INVALID_BASE_ENTRY_DEFINITION.useraction= Specify a valid base entry definition in the server.xml file.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops [hunk 5]
```diff
@@ -186,9 +186,9 @@ ENTITY_HAS_DESCENDENTS=CWIML4519E: The user registry operation could not be comp
 ENTITY_HAS_DESCENDENTS.explanation=The delete or rename function can only be performed on a leaf entity, that is, an entity without any descendants. The specified entity has descendants.
 ENTITY_HAS_DESCENDENTS.useraction=Delete all the  descendants of the entity first, then delete or rename the entity itself.
 
-ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The user registry operation could not be completed. {0} is not valid entity type. Invoke the operation by specifying a valid entity type.
-ENTITY_TYPE_NOT_SUPPORTED.explanation=An operation has been requested on an entity, but that operation does not support the specified entity type. For example, if a repository GET operation is invoked for an Account entity, the operation fails.
-ENTITY_TYPE_NOT_SUPPORTED.useraction=Ensure that operations are invoked for supported entity types and remove any unsupported entity types from the input.
+ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The entity type {0} specified is not a valid entity type for this operation.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The specified entity type {0} is not valid for this operation.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops [hunk 6]
```diff
@@ -186,9 +186,9 @@ ENTITY_HAS_DESCENDENTS=CWIML4519E: The user registry operation could not be comp
...
+ENTITY_TYPE_NOT_SUPPORTED.explanation=An operation has been requested on an entity, but that operation does not support the specified entity type.
+ENTITY_TYPE_NOT_SUPPORTED.useraction=Ensure that the entities provided in the input are of supported entity types, and remove any unsupported entity types from the input.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: ENTITY_TYPE_NOT_SUPPORTED.useraction=Ensure that the entities that are provided in the input are of supported entity types, and remove any unsupported entity types from the input.

**Comment by skasund** [comment]
L2 message review completed

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 5 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 6 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops (hunk 3 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 4 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 2 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops]: Updated.

**Comment by jvanhill** [comment]
#build
#libby

**Review by ramkumar-k-9286 — APPROVED** [review]

**Comment by jvanhill** [comment]
#libby
#build
#spawn.fullfat.buckets=com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1,com.ibm.ws.security.wim.adapter.ldap_fat.krb5.2

**Comment by jvanhill** [comment]
#build

**Review by kristip17 — APPROVED** [review]

**Comment by jvanhill** [comment]
#libby

**Pull request merged**